Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -3534,6 +3536,96 @@ internal static async Task PutObject_Test8(MinioClient minio)
         }
     }
 
+    internal static async Task PutObject_Test9(MinioClient minio)
+    {
+        var startTime = DateTime.Now;
+        var bucketName = GetRandomName(15);
+        var objectName = GetRandomObjectName(10);
+        var contentType = "application/octet-stream";
+        var percentage = 0;
+        var totalBytesTransferred = 0L;
+        var progress = new Progress<ProgressReport>(progressReport =>
+        {
+            percentage = progressReport.Percentage;
+            totalBytesTransferred = progressReport.TotalBytesTransferred;
```

Comment:
just 2 minor cosmetic inputs for `PutObject_Test10`:

1. Increase the size of the test file from `6 * MB` to something like `50 * MB` or something to be able to see more than just 0% and 100% 

2. We can add the progress print and the same output prettification, but commented out.
```
        var progress = new Progress<ProgressReport>(progressReport =>
        {
               percentage = progressReport.Percentage;
               totalBytesTransferred = progressReport.TotalBytesTransferred;
               // Console.WriteLine(
               //    $"Percentage: {progressReport.Percentage}% TotalBytesTransferred: {progressReport.TotalBytesTransferred} bytes");
               // if (progressReport.Percentage != 100) 
               //     Console.SetCursorPosition(0, Console.CursorTop -1);
               // else Console.WriteLine();
        });
```